Functions/saveUserDetailsLocally.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:io';
+import 'dart:convert';
+void saveUSerDetails(var data) {
+  File file = new File("dataBaseUser.txt");
```

Comment:
move this File initialisation in some other FileUtil and make it abstract. So that for any further changes related to file, you need not do it more than one place, introducing redundancy.